### PR TITLE
Misc. typo

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: src/Mod/Fem/femguiobjects/FemSelectionWidgets.py [hunk 1]
```diff
@@ -437,7 +437,7 @@ def selectionParser(self, selection):
                 if solid_to_add:
                     selection = (selection[0], 'Solid' + solid_to_add)
                     ele_ShapeType = 'Solid'
-                    FreeCAD.Console.PrintMessage('selection variable adaped to hold the Solid: ' + selection[0].Shape.ShapeType + '  ' + selection[0].Name + '  ' + selection[1] + ' \n')
+                    FreeCAD.Console.PrintMessage('selection variable adapted to hold the Solid: ' + selection[0].Shape.ShapeType + '  ' + selection[0].Name + '  ' + selection[1] + ' \n')
```
Comment (luzpaz) [inline src/Mod/Fem/femguiobjects/FemSelectionWidgets.py:440]: is it `adapted` or `adopted` ?

**Comment by luzpaz** [comment]
@berndhahnebach also would you mind rebasing the `femdev` branch there are some more recent FEM typos but I don't see them on the current branch.

**Comment by berndhahnebach** [comment]
done.

add them just to the branch of this PR. I will integrate them in my next PR to FreeCAD master all together.

bernd

**Comment by luzpaz** [comment]
> add them just to the branch of this PR. I will integrate them in my next PR to FreeCAD master all together.

Doesn't look like `femdev` branch was rebased yet.

**Comment by luzpaz** [comment]
Rebased and pushed commits. Ready.

**Comment by berndhahnebach** [comment]
https://github.com/FreeCAD/FreeCAD/pull/1723

**Comment by luzpaz** [comment]
@berndhahnebach can you rebase on to master again when you get a chance?

**Comment by berndhahnebach** [comment]
done

**Pull request closed without merging**